modules/Kconfig.tfm

```diff
@@ -5,7 +5,13 @@
 
 menuconfig BUILD_WITH_TFM
 	bool "Build with TF-M as the Secure Execution Environment"
-	depends on TRUSTED_EXECUTION_NONSECURE
+	select CMSIS_RTOS_V2
```

Comment:
@microbuilder @oyvindronningstad I wonder whether these options should be forced here.
I think not, actually. There's no reason for Kernel options like THREAD_STACK_INFO or INIT_STACKS to be selected in the main Kconfig of TFM module. They are sort of un-related.

A compromise would be to switch to "imply", so the user can still override them.

Reply:
Talking about THREAD_STACK_INFO, INIT_STACKS, THREAD_MONITOR, mostly.

Reply:
I'm happy with `imply`. Changed now.

Reply:
Agree with `imply`, thanks.